### PR TITLE
Removes some small straggling redsec items

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Title
- Changes the sec cowboy hats to just be plain black (easier for me)
- Changes the available ushanka to be the blue alt
- Resprites the deputy armband, adds a variant thats available in the loadout (As a bonus, now its less nazi-like to play security, and Lopland has some more ingame representation)
5c8efe2c6024e5e7a1c9d2f5ce62c479-png

Still doesnt touch modsuits because the entire sprite will need redoing (the black-blue it uses is hell) and Im not a spriter

## Changelog

:cl:
fix: Lopland and Armadyne have snagged a few more straggling redsec items, ensuring uniform standards are met.
/:cl:

### CODE REVIEW

**Review by tf-4 — APPROVED** [review]

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm [hunk 1]
```diff
@@ -30,6 +30,17 @@
 	icon_state = "hos_envirohelm_new"
 
 
+////////////////////////////
+//----- ACCESSORIES ------//
+////////////////////////////
```
Comment (tf-4) [inline modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm]:
```
/*
*
*/
```
is a preferable style

**Review by tf-4 — COMMENTED** [review]
Actually just this one thing

File: modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm (hunk 1 above)
Comment (ghost) [reply modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm]: I'd have to change the whole file and Im too lazy for that lmao
This one is clean and more identifiable while you scroll IMO

Comment (ghost) [reply modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm]: ok i just did them all

**Pull request merged**